### ISSUE TITLE
[BUG]: Timeout Error on Subsequent USB TMC Connections to the Oscilloscope.

### ISSUE THREAD

**Issue by u625355** [tektronix/tm_devices#104]
### Description of the bug

Timeout Error on Subsequent USB TMC Connections to the Oscilloscope.

Affected Model: MSO 4 series Oscilloscopes.

The API behaves as expected during the initial connection; however, upon subsequent USB TMC connections to the oscilloscope, an unexpected timeout error occurs, contrasting with the behavior observed in connections through PyVisa.

### Steps To Reproduce

1. Connect the oscilloscope to the system through USB TMC connection.
2. Create a driver object named "scope" for any of the affected models.
3. Use the "scope" driver object query the following API:
   ```
        scope.commands.idn.query()
4. Close the connection.
5. Repeat steps 1-3, observe the error.

### Environment Information

**Operating System:**
- Windows 10, 64-bit

**Software Version:**
-  tm-devices : 0.1.24

**Connection type**
- USB TMC

### Additional Information

Note: 
**This error arises during the execution of consecutive scripts, not in the context of successive API calls.**

**Comment by nfelt14** [comment]
This seems like it may be related to #129

**Comment by u625355** [comment]
This is resolved in version of tm_devices [v(1.2.1)](https://github.com/tektronix/tm_devices/releases/tag/v1.2.1)

Ensure to use **SYSTEM_DEFAULT_BACKEND_VISA** as the visa library of device manager, as Py Visa-PY is not supporting USB connection.
This is already mentioned in official documentation of tm_devices
.[tm_devices: Test & Measurement Device Management — tm_devices documentation (tektronix.github.io)](https://tektronix.github.io/tm_devices/index.html#)
